cuda_phase-retrieval_class/src/example_cuda_no_random.py

```diff
@@ -0,0 +1,44 @@
+import cuPhaseRet #c++ pybind phase retrieval
+import numpy as np
+import imageio
+import matplotlib.pyplot as plt
+import phase_retrieval_python
+import cupy_ref
+from time import perf_counter
+
+#np.random.seed(1)
+image = imageio.imread('../example_images/a.png', as_gray=True)
+array_random = np.random.rand(*image.shape) #uniform random
+mask = np.ones(image.shape) #default mask, must have the same size as input image
+iteration = 100
+
+print("Running phase retrieval with " + str(iteration) + " iterations")
+
+t0_start = perf_counter()
```

Comment:
For performance measurement, only the function `phase_retrieval_pybind.iterate_random_phase(iteration)` should be measured. In the best case, the function `phase_retrieval_pybind.iterate_random_phase(iteration)` was executed one time before without time measurement to initialize the CUDA environment.